library/src/morpheus-master/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/_tables.scss

```diff
@@ -16,6 +16,37 @@ th {
   text-align: left;
 }
 
+//quota reassign dropdown
```

Comment:
Please move this file's(library/src/morpheus-master/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/_tables.scss) changes to library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss

Reply:
Changes have been made